e2e/wdio/core/pages/upload-collection.po.ts

```diff
@@ -0,0 +1,39 @@
+import { CoreBaseComponentPo } from './core-base-component.po';
+import { waitForElDisplayed } from '../../driver/wdio';
+
+export class UploadCollectionPo extends CoreBaseComponentPo {
+    url = '/upload-collection';
+
+    uploadCollectionExample = 'fd-upload-collection-example ';
+    uploadCollectionSmallExample = 'fd-upload-collection-small-example ';
+    uploadCollectionCustomExample = 'fd-upload-collection-custom-example ';
+    uploadCollectionComplexExample = 'fd-upload-collection-complex-example ';
+
+    link = '.fd-link';
+    editButton = '[glyph="edit"]';
+    declineButton = '[glyph="decline"]';
+    input = '.fd-input.ng-star-inserted';
+    okButton = 'button[style="display: inline-flex;"]';
+    item = '.fd-list__item';
+    emphasizedButton = '.fd-button--emphasized';
+    fileUploaderInputFile = 'input.fd-file-uploader__hidden';
+    checkbox = 'fd-checkbox';
+    standardButton = '.fd-button--standard';
+
+    open(): void {
+        super.open(this.url);
+        waitForElDisplayed(this.root);
```

Comment:
for better stability on Safari, please change to 
super.open(this.url);
waitForPresent(this.root)
waitForElDisplayed(this.title);